### DISCUSSION TITLE
Be able to quote and cite a part of the cast

### DISCUSSION THREAD

**Discussion by nicopace** [asciinema/discussions#106] [Ideas]
When you are having a conversation about a particular cast (let's say, on github, or any platform where you can embed markdown), it would be great if the platform would provide a way of building an embeddable quote of your current selection of the console, with a link to the specific moment that you are looking at.

For example, for the one that is used in the website: https://asciinema.org/a/335480

It would be great if the share would highlight the selected text, like this:
[![asciicast](https://user-images.githubusercontent.com/1136597/105764483-c781a580-5f35-11eb-85eb-53faaeb75d02.png)](https://asciinema.org/a/335480?t=76)

Would help peer-programming communities in debating about what has been done in a session.

**Comment by ku1ik** [comment]
This sounds really cool!
